plugin/documents.py

```diff
@@ -490,8 +490,10 @@ def _on_code_actions(self, responses: CodeActionsByConfigName) -> None:
                 title = '{} code actions'.format(action_count)
             else:
                 title = next(itertools.chain.from_iterable(responses.values()))['title']
+                title = "\n".join([title[:30], title[30:]])
             code_actions_link = make_command_link('lsp_code_actions', title, {"commands_by_config": responses})
-            annotations = ["<div class=\"actions\">{}</div>".format(code_actions_link)]
+            annotations = [
+                "<div class=\"actions\" style=\"width:100%;font-family:system\">{}</div>".format(code_actions_link)]
```

Comment:
Can you please make these changes in a separate pull request? This conflates two topics. Also, use `textwrap.wrap`, as the current code might split at non-word boundaries.

Reply:
Reso